### PR TITLE
Degree Days dataset SNAP portal prep

### PR DESCRIPTION
**Pull request by kyleredilla**
This PR adds a single Jupyter notebook (`arctic_eds/degree_days/portal_prep.ipynb`) for extracting metadata from and zipping the degree days data (as generated in this folder; no design indices in this one) for hosting via the SNAP GeoNetwork data portal ([here](https://catalog.snap.uaf.edu/geonetwork/srv/eng/catalog.search#/metadata/3d4cc2e2-5cfb-4fed-b397-06854edb747f)). This also includes a `zipit.sh` script for helping to zip the data files. To review, feel free to just have a look without running, or run the whole notebook. Any feedback is welcome! No rush on this.

### CODE REVIEW

File: arctic_eds/degree_days/portal_prep.ipynb [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "id": "5ef95df6-70a2-42c5-a611-5bfa8ed16a18",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import subprocess\n",
+    "from pathlib import Path\n",
```
Comment (charparr) [inline arctic_eds/degree_days/portal_prep.ipynb:22]: I think the import of `CRS` went missing, I added it back in like so: `from rasterio.crs import CRS
` and then ran the notebook successfully!

File: arctic_eds/degree_days/zipit.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+directory="$1"
+dd_name="$2"
+zip_directory="$3"
+
+zip_file="${zip_directory}/${dd_name}.zip"
+
+echo "Creating ${zip_file}"
+# delete zip archive if it exists already
+rm -f "${zip_file}"
```
Comment (charparr) [inline arctic_eds/degree_days/zipit.sh:11]: When I executed this script via the notebook subprocess I hit a "permission denied" error which makes sense:
`rm: cannot remove ‘/workspace/Shared/Tech_Projects/Arctic_EDS/project_data/dd_zips/thawing_index.zip’: Permission denied`

Not a big deal, and maybe a good fail safe!

I'd say leave it as is, but if you really feel like fiddling with it you could reconfigure the zip output directory to be more local to the user, so they can zip in their own space and then copy/overwrite what is in `Shared/Tech_Projects` on their own.

**Review by charparr — CHANGES_REQUESTED** [review]
Thanks for tackling this Kyle! I love the idea of a "portal_prep" notebook being part of our ingest "pipeline" - it makes a ton of sense and should save us work in the future when people ask us for data that we are serving via Rasdaman. May as well expose it on the GeoNetwork too :) 

Only change I am requesting is the missing`CRS` import in the notebook.

File: arctic_eds/degree_days/zipit.sh (hunk 2 above)
Comment (kyleredilla) [reply arctic_eds/degree_days/zipit.sh:11]: I support this philosophy! Implemented.

**Review by charparr — APPROVED** [review]

**Comment by charparr** [comment]
Thanks Kyle, looks good! Note that I didn't re-run with this small change, but I inspected the #[58ff48c](https://github.com/ua-snap/rasdaman-ingest/pull/62/commits/58ff48c0079c352e913d33ba97062d7105d32b5e) diff.

**Pull request merged**